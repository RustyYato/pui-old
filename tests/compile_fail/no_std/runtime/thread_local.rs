use pui::make_counter;

make_counter! {
    threadlocal type _Runtime = u8;
}

fn main() {}
