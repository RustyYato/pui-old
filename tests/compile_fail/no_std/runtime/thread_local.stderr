error: the `std` feature on `pui` msut be turned on to allow thread local storage
 --> $DIR/thread_local.rs:3:1
  |
3 | / make_counter! {
4 | |     threadlocal type _Runtime = u8;
5 | | }
  | |_^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
